share/locale/en.json

```diff
@@ -1,165 +1,165 @@
 {
-    "config_onion_service": "Configuring onion service on port {0:d}.",
-    "preparing_files": "Preparing files to share.",
+    "config_onion_service": "Setting up onion service on port {0:d}.",
+    "preparing_files": "Preparing OnionShare.",
     "wait_for_hs": "Waiting for HS to be ready:",
     "wait_for_hs_trying": "Trying…",
     "wait_for_hs_nope": "Not ready yet.",
-    "wait_for_hs_yup": "Ready!",
-    "give_this_url": "Give this address to the person you're sending the file to:",
-    "give_this_url_stealth": "Give this address and HidServAuth line to the person you're sending the file to:",
-    "give_this_url_receive": "Give this address to the people sending you files:",
-    "give_this_url_receive_stealth": "Give this address and HidServAuth line to the people sending you files:",
+    "wait_for_hs_yup": "Ready.",
+    "give_this_url": "Give this address to the recipient of the file:",
+    "give_this_url_stealth": "Give this address and HidServAuth line to the recipient of the file:",
+    "give_this_url_receive": "Give this address to those that send you files:",
+    "give_this_url_receive_stealth": "Give this address and your HidServAuth line to those that send you files:",
     "ctrlc_to_stop":  "Press Ctrl+C to stop the server",
     "not_a_file": "{0:s} is not a valid file.",
     "not_a_readable_file": "{0:s} is not a readable file.",
-    "no_filenames": "You must specify a list of files to share.",
-    "no_available_port": "Could not start the Onion service as there was no available port.",
-    "download_page_loaded": "Download page loaded",
-    "other_page_loaded": "Address loaded",
```

Comment:
These two seem more like debugging than conveyors of actual info.

Reply:
When you start OnionShare in share mode, it runs a full website on your computer. So if your OnionShare address is `http://[blah].onion/subway-oxidant`, then there are actually several different addresses that could be loaded.

Someone could load `http://[blah].onion/subway-oxidant/download`, which will right away download the files. But someone could load just `http://[blah].onion/` which will show them a 404 error page and not let them have access -- and this is true if they load any other 404 page like `http://[blah].onion/trying-to-guess-the-slug`.

Basically, when you share an OnionShare address, it's nice to know when the person loads the download page, actually starts to download the files, or if anyone is loading any other addresses on the website -- which could be a sign of someone trying to attack it. I think we should probably keep these strings.

Reply:
How about "OnionShare found". or "OnionShare opened"?